notebooks/lesson07_classification.ipynb

```diff
@@ -0,0 +1,3932 @@
+{
```

Comment:
<p>this plot does not seem very informative. maybe delete it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='105'/>

Reply:
<p>done!</p>